static/js/stream_edit.js

```diff
@@ -219,6 +219,11 @@ function show_subscription_settings(sub_row) {
             return item.email;
         },
     });
+    if (people.human_realm_count() -
+        stream_data.get_human_subscriber_count(sub.subscribers) === 0) {
```

Comment:
This code feels brittle to me.  We're trying to figure out if there any unsubscribed user in our realm, because otherwise there's no point trying to add users, correct?  Well, we should just calculate that set of user_ids, because don't we want to use the same users in typeahead?  And then once we have that `Set` of user_ids, we can use `.size` on the set here.

Reply:
Perhaps a better helper function here would be in `stream_data` and would be something like `get_unsubscribed_humans_for_stream(stream_id)`.

Reply:
Should I retain the functions I've created in the first 2 commits? As far as I've observed and discussed in the community, those functionalities don't exist.

Reply:
I'm working on a PR to clean up the people/stream_data APIs here.

Reply:
Once I finish that, I'll let you know and you can cherry-pick my commits into your PR and probably remove one or two of your commits.

Reply:
See #14280.  I think you actually only need the second commit there, which will give you `stream_data.potential_subscribers()`.  Note that I think you need to be careful about which function you call this in--you want to be sure that we update the messaging any time new subscriber events come in.

I don't believe that you'll have any need for my first commit, which adds `people.get_active_human_count`, but it's obviously relevant to some conversations we've had related to this PR.

Reply:
Thanks. I'll take a look.